main.rb

```diff
@@ -0,0 +1,12 @@
+require './app'
+
+def main(app = App.new)
+  input = app.query_method
+  if input <= 6 && input >= 0
+    app.return_method(input)
+  else
+    app.return_error
+  end
+
+  main(app) unless input == 7
+end
```

Comment:
- [ ] kindly call the method `main` at the end of this file so that the next reviewer can run this app in the console by running `ruby main.rb`

Reply:
@VICTOR-CHUKWU I have carried out all of the required changes.